### PR TITLE
Makes all of the NSV floor tiles one type of durasteel tile

### PR DESCRIPTION
**Pull request by Bokkiewokkie**
## About The Pull Request

Makes all floor tiles used on NSV have a material datum and craftable from durasteel as one unified tile, also adds the ability to build plating from said durasteel tiles. This lets you stack multiple different kinds of types of tile onto one stack and change the type of tile with Ctrl+click.

Todo:
- [x] make sure dragging them doesn't open the menu
- [x] make canceling the selection not change the icon to nothing

## Why It's Good For The Game

These tiles are a pain in the ass to use for actual construction, this makes them a lot more user friendly.
Requested by @Vasily2013 among other people.

## Changelog
:cl:
tweak: changed durasteel floor tiles to allow changing their type by Ctrl-clicking them
fix: fixed not being able to make plating with durasteel tiles
imageadd: added floor tile sprites for monotile floors
refactor: changed ship/durasteel tiles to all stack with eachother
/:cl:

### CODE REVIEW

**Comment by Bokkiewokkie** [comment]
> This pull request has conflicts, please resolve those before we can evaluate the pull request.

Shhhh you don't know what you're talking about shut up

**Review by Karmic-Skink — DISMISSED** [review]

**Comment by benbot16** [comment]
Ctrl-clicking the tiles broke on round 4677, cap couldn't change the tiles to different types.

File: nsv13/code/game/objects/items/custom_materials.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 //Metals
 
-GLOBAL_LIST_INIT(duranium_recipes, list (\
+GLOBAL_LIST_INIT(duranium_recipes, list ( \
 	new/datum/stack_recipe("railgun rail", /obj/item/ship_weapon/parts/railgun_rail, 1, time = 20, one_per_turf = FALSE, on_floor = TRUE), \
 	new/datum/stack_recipe("mac barrel", /obj/item/ship_weapon/parts/mac_barrel, 1, time = 20, one_per_turf = FALSE, on_floor = TRUE), \
```
Comment (Karmic-Skink) [inline nsv13/code/game/objects/items/custom_materials.dm:5]: Forgot these were still in here.

**Review by Karmic-Skink — CHANGES_REQUESTED** [review]
Could you please update Duranium and Durasteel to roughly match APNW requirements.

Durasteel Ratio: Iron 0.2, Silver 0.15, Titanium 0.65.
Duranium Ratio: Iron 0.175, Silver 0.15, Plasma 0.05, Titanium 0.625.

File: nsv13/code/game/objects/items/custom_materials.dm (hunk 1 above)
Comment (Bokkiewokkie) [reply nsv13/code/game/objects/items/custom_materials.dm:5]: I believe we do still use some of these in construction of deck guns

**Review by Karmic-Skink — APPROVED** [review]

**Comment by Bokkiewokkie** [comment]
Time to watch the ~~fireworks~~ merge conflicts

**Pull request merged**